app/dashboard/views.py

```diff
@@ -403,3 +403,95 @@ def prirp(request):
 
 def apitos(request):
     return redirect('https://gitcoin.co/terms#privacy')
+
+def toolbox(request):
+    actors = [{
+        "title": "Developers",
+        "description": "Solve issues. Collect bounties!",
+        "tools": [{
+            "name": "Issue Explorer",
+            "img": "/static/v2/images/why-different/code_great.png",
+            "description": '''A searchable index of all of the work available in
+                            the system.''',
+            "link": "https://gitcoin.co/explorer",
+            "active": "true"
+        }, {
+            "name": "API",
+            "img": "/static/v2/images/tools/api.jpg",
```

Comment:
can we make these images more performant?  right now api.jpg is almost a MB.. and it probably shouldnt be

Reply:
Updated the images ^_^

<img width="622" alt="screen shot 2017-12-28 at 4 44 27 pm" src="https://user-images.githubusercontent.com/5358146/34426721-746e5414-ebee-11e7-9151-b5233f156951.png">